<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CsvImportWizardPage4</class>
 <widget class="QWizardPage" name="CsvImportWizardPage4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Es konnten nicht alle Artikel aufgrund Ungereimtheiten importiert werden.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="infoLabel">
       <property name="text">
        <string>Information</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Ausgewähle:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="newImportButton">
         <property name="toolTip">
          <string>Artikel werden mit neuer Artikelnummerübernommen.</string>
         </property>
         <property name="text">
          <string>Als neuen Artikel Importieren</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="changeImportButton">
         <property name="toolTip">
          <string>Bestehende Artikel werden mit den neuen Daten überschrieben</string>
         </property>
         <property name="text">
          <string>Ändern</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QrkPushButton" name="cancelImportButton">
         <property name="toolTip">
          <string>Diese Artikel werden nicht Importiert</string>
         </property>
         <property name="text">
          <string>Verwerfen</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QrkPushButton</class>
   <extends>QPushButton</extends>
   <header>qrkpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
